assets/javascripts/discourse/initializers/chat-setup.js

```diff
@@ -78,6 +78,25 @@ export default {
       );
 
       api.addToHeaderIcons("header-chat-link");
+
+      api.decorateChatMessage(function (chatMessage) {
+        if (!this.currentUser) {
+          return;
+        }
+
+        const highlightable = [
+          `@${this.currentUser.username}`,
+          "@here",
+          "@all",
+        ];
```

Comment:
Btw this means that other users won't be highlighted?

Reply:
Also noted that `here` and `all` are keywords but hardcoded here.

Reply:
Yes I too find the naming a little bit weird as ".mention" will already be highlighted and ".highlighted" is basically "even more highlighting".

This is out of the scope of this PR but we could maybe refactor to have better naming here.

Reply:
> Also noted that `here` and `all` are keywords but hardcoded here.

what do you suggest instead?

Reply:
> as ".mention" will already be highlighted and ".highlighted" is basically "even more highlighting"

Got it.

> what do you suggest instead?

Personally I was thinking something along the lines of `MENTION_KEYWORDS = ["here", "all"]`, mostly because in some other places like slack, there's also `channel` and potentially other configurable terms/aliases.

Reply:
Works for me, will do the change.